packages/styles/scss/components/audio-player/_audio-player.scss

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../globals/imports';
+@import '../../globals/utils/ratio-base';
+@import 'carbon-components/scss/components/slider/slider';
+
+@mixin audio-player {
+  .#{$prefix}--audio-player {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 48px;
+    padding: 0 8px 0 0;
+    background-color: $ui-01;
+
+    &__captions {
```

Comment:
looks great!! Just to add one small thing to @kennylam's comments. Wondering if we can avoid concatenation in the styles. For example, `.#{$prefix}--audio-player__captions` instead. This will help us in the future when building the web component version